bootstrap/pkg/apis/apps/configconverters/v1alpha1.go

```diff
@@ -61,13 +61,14 @@ func (v V1alpha1) ToKfConfig(appdir string, kfdefBytes []byte) (*kfconfig.KfConf
 	}
 
 	config := &kfconfig.KfConfig{
```

Comment:
Why doesn't KFConfig have a Spec and a Status?

Reply:
done